bundle-workflow/scripts/default/build.sh

```diff
@@ -13,10 +13,11 @@ function usage() {
     echo -e "-s SNAPSHOT\t[Optional] Build a snapshot, default is 'false'."
```

Comment:
Move `scripts/default/build.sh` into `scripts/OpenSearch/default` and `scripts/OpenSearch-Dashboards/default` and call the right one, same with install.